src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card {
+  max-width: 200px;
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid white;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0;
```

Comment:
Make like a `reset` file, remove default value before your css classes

Reply:
do it in all cases

Reply:
Yes, you can do something like reset the file, you can copy here, but it's extra
you can just reset some margins, for expample:
```
h1, 
h2, 
h3, 
... 
p, 
ul {
  margin: 0
}
```
& paddings 
```
ul, 
li, 
... {
  padding: 0;
}
```